src/main/java/com/walmartlabs/x12/standard/txset/generic/GenericTransactionSet.java

```diff
@@ -77,6 +77,35 @@ public void addX12Loop(X12Loop loop) {
         loops.add(loop);
     }
     
+    @Override
+    public boolean hasLooping() {
+        // TODO: Generic will have no way of knowing
```

Comment:
🤔 
opens up question is there value to this method...
without it user would just have to work w/ the List of X12Loop(s) 
but in most non-generic cases the top of the loop will be a subclass (see Shipment on ASN)

might be best to assume that when non-generic (like ASN) that users will know that there are loops (or not like in the case of a PO). 

And then assume in the case of generic that a user will have to do something like this:

```
if (!CollectionUtils.isEmpty(txSet.getLoops()) && txSet.isLoopingValid()) {
   // we have loops to deal with
} else if (!CollectionUtils.isEmpty(txSet.getLoopingErrors())) {
  // we have loop errors to deal with
}
```